derp/derphttp/derphttp_server.go

```diff
@@ -20,6 +20,11 @@ const fastStartHeader = "Derp-Fast-Start"
 
 func Handler(s *derp.Server) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.Method == http.MethodGet && r.URL.Path == "/stat" {
```

Comment:
put this in cmd/derper/derper.go's `func debugHandler` instead.

Make it be, say, `/debug/traffic`, and have the few lines in derper.go's just be like:

```go
                if r.RequestURI == "/debug/traffic" {
                       s.ServeDebugTraffic(w, r)
                       return
                }
```

Then add a new that new `ServeDebugTraffic` to derp/derp_server.go (not derphttp_server.go).